docs/simulate.ron

```diff
@@ -267,6 +267,8 @@
             ),
             /* shape (u) for the Clark 2Dt dispersal kernel */
             shape_u: (0.0 < f64),
+            /* tail (p) for the Clark 2Dt dispersal kernel */
```

Comment:
Can we add a note stating that  the 2Dt tends to Gaussian as p becomes large and tends to Cauchy as p goes to zero? Also could be have the default value for the p parameter be 1?